catalog/app/components/Preview/renderers/Perspective/types.ts

```diff
@@ -0,0 +1,14 @@
+import type * as React from 'react'
+
+import type { S3HandleBase } from 'utils/s3paths'
+
+import { ParquetMetadata } from '../../loaders/Tabular'
+
+export interface PerspectiveProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
*probably* this can be kept in `Perspective.tsx` and imported as `import type ...` in `index.ts` (type import *should not* affect code-splitting) to avoid creating an extra file and moving definition away from the code using it, but it's not a big deal

Reply:
I thought about that but didn't try :)
Just a minute

Reply:
Seems like it works